src/AppKit/NSAccessibility.cs

```diff
@@ -80,6 +89,93 @@ public static CGPoint GetPointInView (NSView parentView, CGPoint point)
 		{
 			return NSAccessibilityPointInView (parentView, point);
 		}
+
+		[Mac (10,7)]
+		[DllImport (Constants.AppKitLibrary)]
+		static extern void NSAccessibilityPostNotificationWithUserInfo (IntPtr element, IntPtr notification, IntPtr userInfo);
+
+		public static void PostNotification (NSObject element, NSString notification, NSDictionary userInfo)
+		{
+			NSAccessibilityPostNotificationWithUserInfo (element.Handle, notification.Handle, userInfo.Handle);
+		}
+
+		[DllImport (Constants.AppKitLibrary)]
+		static extern void NSAccessibilityPostNotification (IntPtr element, IntPtr notification);
+
+		public static void PostNotification (NSObject element, NSString notification)
+		{
+			NSAccessibilityPostNotification (element.Handle, notification.Handle);
+		}
+
+		[DllImport (Constants.AppKitLibrary)]
+		static extern IntPtr NSAccessibilityRoleDescription (IntPtr role, IntPtr subrole);
+
+		public static string RoleDescription (NSString role, NSString subrole)
+		{
+			IntPtr handle = NSAccessibilityRoleDescription (role.Handle, subrole.Handle);
+			return NSString.FromHandle (handle);
+		}
+
+		[DllImport (Constants.AppKitLibrary)]
+		static extern IntPtr NSAccessibilityRoleDescriptionForUIElement (IntPtr element);
+
+		public static string RoleDescription (NSObject element)
+		{
+			IntPtr handle = NSAccessibilityRoleDescriptionForUIElement (element.Handle);
+
+			if (handle == IntPtr.Zero)
+				return null;
+
+			return NSString.FromHandle (handle);
+		}
+
+		[DllImport (Constants.AppKitLibrary)]
+		static extern IntPtr NSAccessibilityActionDescription (IntPtr action);
+
+		public static string ActionDescription (NSString action)
+		{
+			IntPtr handle = NSAccessibilityActionDescription (action.Handle);
+			return NSString.FromHandle (handle);
+		}
+
+		[DllImport (Constants.AppKitLibrary)]
+		static extern IntPtr NSAccessibilityUnignoredAncestor (IntPtr element);
+
+		public static NSObject UnignoredAncestor (NSObject element)
+		{
+			var handle = NSAccessibilityUnignoredAncestor (element.Handle);
+			return new NSObject (handle);
```

Comment:
Use `Runtime.GetNSObject (handle)` instead, which does not create a new managed wrapper if one already exists.